modules/gke-cluster/variables.tf

```diff
@@ -58,5 +58,5 @@ variable "initial_node_count" {
 }
 
 variable "kubernetes_version" {
-  default = "1.9.7-gke.1"
+  default = "1.10.5-gke.0"
```

Comment:
New version of k8s. I don't mind trying it, and don't think that we're doing to face any issues because of upgrade. GKE default version is still 1.9.7-gke.3 though.